pages/common/chisel.md

```diff
@@ -0,0 +1,28 @@
+# chisel
+
+> Chisel is a fast and simple tool to create a TCP tunnel. Including both client and server.
```

Comment:
We tend to omit adjectives ("fast", "simple", "top quality") in the main description. Stick to describing what it is and what it does.